cypress/e2e/19-execution.cy.ts

```diff
@@ -569,4 +569,27 @@ describe('Execution', () => {
 			expect(interception.request.body.runData).to.include.all.keys(expectedRunDataKeys);
 		});
 	});
+
+	it('should successful execute partial executions with nodes attached to the second output', () => {
```

Comment:
Not the best name, happy to take suggestions. Anything else I come up with is equally convoluted. 😅

Reply:
I totally feel with you. Programming is easy, but naming things is hard.

Reply:
I fixed the adverb at least 😅